### PR TITLE
fix(tag-editor) cast to boolean

### PR DESCRIPTION
**Pull request by jackkav**
Removes the janky 0 from the missing bool cast on the length check, thanks React for believing in zeros.

image

🪳 🔫 

image

### CODE REVIEW

File: packages/insomnia-app/app/ui/components/templating/tag-editor.tsx [hunk 1]
```diff
@@ -767,12 +767,11 @@ class TagEditor extends PureComponent<Props, State> {
...
-        {activeTagDefinition &&
-          activeTagDefinition.args.map((argDefinition: NunjucksParsedTagArg, index) =>
-            this.renderArg(argDefinition, activeTagData.args, index),
-          )}
+        {activeTagDefinition?.args.map((argDefinition: NunjucksParsedTagArg, index) =>
+          this.renderArg(argDefinition, activeTagData.args, index),
+        )}
 
-        {activeTagDefinition?.actions?.length && this.renderActions(activeTagDefinition.actions)}
+        {!!activeTagDefinition?.actions?.length && this.renderActions(activeTagDefinition.actions)}
```
Comment (dimitropoulos) [inline packages/insomnia-app/app/ui/components/templating/tag-editor.tsx]:
```suggestion
        {activeTagDefinition?.actions?.length > 0 ? this.renderActions(activeTagDefinition.actions) : null}
```

There's no way you could know this but we're trying to get away from the `&&` pattern for precisely this reason (and more).  Here we can just use a ternary instead.

Comment (jackkav) [reply packages/insomnia-app/app/ui/components/templating/tag-editor.tsx]: Appreciate the assist!

**Review by dimitropoulos — APPROVED** [review]

**Comment by jackkav** [comment]
First merged insomnia PR heck yeah!

**Comment by dimitropoulos** [comment]
language, jack, language!  we're a family-friendly oss!

**Comment by develohpanda** [comment]
Thanks for fixing this; it's an itch that has been around but we haven't been able to get to 🙌🏽

**Pull request merged**